run.py

```diff
@@ -21,10 +22,16 @@ def main():
 
     init_rabbitmq()  # test the connection to the rabbitmq cluster
 
-    subscriber_future = setup_subscription()
-    with Readiness(os.getenv('READINESS_FILE_PATH', os.path.join(os.getcwd(), 'pubsub-ready'))):  # Indicate ready after successful setup
+    futures = [setup_subscription(),
+               setup_subscription(subscription_name=OFFLINE_SUBSCRIPTION_NAME, callback=offline_receipt_to_case,
+                                  subscription_project_id=OFFLINE_SUBSCRIPTION_PROJECT_ID)]
+
+    with Readiness(os.getenv('READINESS_FILE_PATH',
+                             os.path.join(os.getcwd(), 'pubsub-ready'))):  # Indicate ready after successful setup
+        for subscriber_future in futures:
+            thread = threading.Thread(target=subscriber_future.result, kwargs={"timeout": None}, daemon=True)
+            thread.start()
```

Comment:
This doesn't cope with the threads dying or not starting for any reason, for example you can start the app up in an environment with no pubsub topics/subscriptions and it will start, show ready and enter the infinite loop below even though it never subscribed to anything. Also if either thread dies for any reason the app will just carry on running in that while loop.

Reply:
Should probably call `thread.join()` to break out the Readiness context when a thread exits. And I'm not sure the while loop is needed below - unless there's an outer infinite loop that restarts the threads when they die...?

Reply:
@jcox-dev But wouldn't thread.join() only let it join one of the two thread, since it's essentially synchronously waiting on that thread from that point?

Reply:
ch ch ch check out gav's latest changes innit

Reply:
works for me 🤷‍♂